source/state_representation/src/space/cartesian/CartesianState.cpp

```diff
@@ -15,30 +15,22 @@ CartesianState::CartesianState(const std::string& robot_name, const std::string&
   this->initialize();
 }
 
-CartesianState::CartesianState(const CartesianState& state) : SpatialState(state),
-                                                              position(state.position),
-                                                              orientation(state.orientation),
-                                                              linear_velocity(state.linear_velocity),
-                                                              angular_velocity(state.angular_velocity),
-                                                              linear_acceleration(state.linear_acceleration),
-                                                              angular_acceleration(state.angular_acceleration),
-                                                              force(state.force),
-                                                              torque(state.torque) {}
+CartesianState::CartesianState(const CartesianState& state) = default;
```

Comment:
Same comment here, I think the `default` directive is nicer to see in the header. If you do that, the header (and Doxygen generation) will show that the copy constructor will have the behaviour. If you only change the source, there is now way to know just from the interface that the behaviour may be different.